src/components/Table/Example.md

```diff
@@ -417,3 +417,26 @@ const data = [{
 
 <Table data={data} groupBy="name" />
 ```
+
+** onSort Callback **
+
+Providing an `onSort` callback allows you to do additional data fetching if the user invokes sorting on your table.
+
+```jsx { "props": { "data-skip": true } }
+const { Table } = require('precise-ui');
+const data = [{
+  name: 'A',
+  age: 21,
+}, {
+  name: 'B',
+  age: 12,
+}, {
+  name: 'C',
+  age: 25,
+}, {
+  name: 'D',
+  age: 15,
+}];
+
+<Table data={data} onSort={(ev) => console.log(ev)} />
```

Comment:
Yeah, because the whole sorting is disable when `onHeaderClick` is present.

Reply:
Did you figure out the possible way to control the table sorting with `onHeaderClick` and `sortBy` props? 

I'm not 100% sure, but I guess it's possible to do it using them.

Reply:
Rough example for you case can be:
- pass initial `data={data}` to the component
- add event handler `onHeaderClick={fetchData}` (`fetchData = (sortBy?: SortingType) { ... }`)
- change value of `sortBy={sortBy}` within fetchData function

Reply:
That's definitely possible and I gave it a try. In the end I was rebuilding most of the default `TableBasic.part.tsx` internals and figured it would make more sense to be able to pass a `onSort` callback into the table.

Reply:
> Yeah, because the whole sorting is disabled when `onHeaderClick` is present.
It would be good to have it documented(as a _JSDoc_) for `onSort` type.

Reply:
> Rough example for you case can be:
> 
> * pass initial `data={data}` to the component
> * add event handler `onHeaderClick={fetchData(sortBy)}` (`fetchData = (sortBy?: SortingType) { ... }`)
> * change value of `sortBy={sortBy}` inside _fetchData_ function

This would require to reimplement the whole [`getDefaultHeaderCellRenderer`](https://github.com/ZEISS/precise-ui/blob/86be817ae4e6ca08a996f941309126851c79d0b1/src/components/Table/TableShared.part.tsx#L62) on top of the sorting in the host component which I wanted to avoid.

Reply:
Sounds like a headache 😟